com.github.onegal.Viper.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: com.github.onegal.Viper
```

Comment:
It should be:

```suggestion
app-id: com.github._0negal.Viper
```


see https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic

Reply:
Would manifest filename etc then be `com.github._0negal.Viper.yaml` or `com.github.0negal.Viper`? ^^

Reply:
yes

Reply:
Done with 78de2a6e76af3d9ed929651bef253dfedf2935df